sudoku/easy/script.js

```diff
@@ -0,0 +1,53 @@
+import { addRandom } from './addRandom.js';
+import { checkGrid, checkRowsArr, checkGridArr, checkColumnsArr } from './checkFunctionsRandom.js';
+import { start, pause, reset } from '../general/stopwatch.js';
+export let answers = [];
+export let check = document.querySelector('.check');
+let div = document.querySelectorAll('.board>div');
+export let startGamebtn = document.querySelector('.change');
+let matrix = [];
+let arr = [];
+
+for (let i = 0; i < div.length; i++) {
+  div[i].classList.add(i + 1);
+}
+////emptying
+startGamebtn.addEventListener('click', () => {
+  for (let i = 0; i < div.length; i++) {
+    div[i].innerHTML = '';
+  }
+});
```

Comment:
instead of writing a comment, you could separate the emptying logic into a separate function named `emptyBoard()` and call it here.